call.go

```diff
@@ -165,7 +164,11 @@ func Invoke(ctx context.Context, method string, args, reply interface{}, cc *Cli
 			return toRPCErr(err)
 		}
 		if EnableTracing {
-			c.traceInfo.tr.LazyLog(payload{args}, true)
+			p := &payload{
```

Comment:
there's no need for these to be separate vars. they are tiny. inline it in the LazyLog calls.